Sources/XCTest/Public/XCTestMain.swift

```diff
@@ -12,14 +12,20 @@
 //  for running tests and some infrastructure for running them.
 //
 
+#if os(macOS)
+    import SwiftFoundation
+#else
+    import Foundation
+#endif
+
 #if os(Linux) || os(FreeBSD)
     import Glibc
-    import Foundation
-#else
+#elseif os(macOS)
     import Darwin
-    import SwiftFoundation
 #endif
 
+#if os(macOS) || os(Linux) || os(FreeBSD)
```

Comment:
@briancroom thank you for pointing this out. I thought that `func XCTMain` won't compile for iOS (e.g. since exit method usage), but I was wrong. I just checked and it compiles. So I removed this condition.